Mathlib/NumberTheory/Cyclotomic/Basic.lean

```diff
@@ -3,6 +3,7 @@ Copyright (c) 2021 Riccardo Brasca. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Riccardo Brasca
 -/
+import Mathlib.Init.Core
```

Comment:
Why is this so?

Reply:
Added imports happen because the import was removed from an upstream file that didn't need it, and downstream files were accidentally relying on the transitive import.

Reply:
I mean, I don't think this file (or any file) uses `Mathlib.Init.Core`. But after more inspection it seems like that file contains a single simp attribute that was maybe used.